### DISCUSSION TITLE
Correct notation for defining variables

### DISCUSSION THREAD

**Discussion by felixcrazzolara** [idaholab/moose#26435] [Q&A General] [answered]
I am a bit confused about the correct notation for the definition of variables. For example here
https://mooseframework.inl.gov/getting_started/examples_and_tutorials/examples/ex01_inputfile.html
variables are defined as
[./variable_name]
...
[../]
but in most other places, they are defined as follows
[variable_name]
...
[]

What's the difference between the two ways to define variables?

**Comment by GiudGiud** [comment, marked as answer]
Hello

Those are the same in MOOSE.
The ../ and ./ are a legacy syntax but it wont error or complain if you use it.

Guillaume
